go/private/repositories.bzl

```diff
@@ -17,7 +17,7 @@
 load("@io_bazel_rules_go//go/private:toolchain.bzl", "go_sdk_repository", "go_host_sdk_repository")
 load("@io_bazel_rules_go//go/private:repository_tools.bzl", "go_repository_tools")
 load("@io_bazel_rules_go//go/private:go_repository.bzl", "go_repository")
-load('@io_bazel_rules_go//go/toolchain:toolchains.bzl', 'register_go_toolchains')
+load('@io_bazel_rules_go//go/toolchain:toolchains.bzl', 'go_register_toolchains')
```

Comment:
Nit: double-quotes for consistency with other statements.

Reply:
Done